src/open_inwoner/configurations/models.py

```diff
@@ -19,7 +19,7 @@
 from ..utils.css import clean_stylesheet
 from ..utils.fields import CSSField
 from ..utils.files import OverwriteStorage
-from ..utils.validators import FilerExactImageSizeValidator
+from ..utils.validators import DutchPhoneNumberValidator, FilerExactImageSizeValidator
```

Comment:
Hopefully this validator also accepts the short phonenumbers municipalities now use? Like Groningen only uses "14 050" without an area code etc.